packit_service/constants.py

```diff
@@ -1,6 +1,10 @@
 from enum import Enum
 
-FAQ_URL = "https://packit.dev/packit-as-a-service/#faq"
+DOCS_URL = "https://packit.dev/packit-as-a-service/"
+FAQ_URL = f"{DOCS_URL}#faq"
+FAQ_URL_HOW_TO_RETRIGGER = (
+    DOCS_URL + "#how-to-re-trigger-packit-service-actions-in-your-pull-request"
+)
```

Comment:
I know you hate this kind of review comments, but, why the concatenation?
```suggestion
FAQ_URL_HOW_TO_RETRIGGER = f"{DOCS_URL}#how-to-re-trigger-packit-service-actions-in-your-pull-request"
```

Reply:
¯\_(ツ)_/¯

Reply:
because the line is too long :P you nitpicker